sdk/core/amqp-common/package.json

```diff
@@ -106,11 +108,11 @@
     "lint": "eslint -c ../../.eslintrc.json src test samples --ext .ts -f node_modules/eslint-detailed-reporter/lib/detailed.js -o amqp-common-lintReport.html || exit 0",
     "pack": "npm pack 2>&1",
     "prebuild": "npm run clean",
-    "test:browser": "npm run build:test && npm run unit-test:browser && npm run integration-test:browser",
-    "test:node": "npm run build:test && npm run unit-test:node && npm run integration-test:node",
-    "test": "npm run build:test && npm run unit-test && npm run integration-test",
+    "test:browser": "npm run build:test:browser && npm run integration-test:browser",
+    "test:node": "npm run build:test:node && npm run integration-test:node",
```

Comment:
In https://github.com/Azure/azure-sdk-for-js/pull/5110#discussion_r323510035, I had suggested that the `test:*` scripts should run the unit tests and not integration tests. Any reason to not do so?

The `test:*` scripts are for the developer's convenience. It doesn't get used by PR CI or the nightly CI. This is purely meant for the dev's inner loop.

In the case of amqp-common, since we have only the unit tests, its that we should target in the `test:*` scripts.